microsoft-edge/webview2/concepts/print.md

```diff
@@ -0,0 +1,736 @@
+---
+title: Printing in WebView2 apps
+description: How to print a web page in a WebView2 app.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 01/12/2023
+---
+# Print
+
+There are a few different ways to print a web page in WebView2 which give you varying levels of ease of use and control.
+
+## ShowPrintUI
+
+The ShowPrintUI show the WebView2 print preview dialog or OS print dialog for the current top-level document in the WebView2. With this mechanism you easily get a printing experience the user will be familiar with.
+
+<!-- ------------------------------ -->
+
```

Comment:
GLOBAL: for link text that needs to include params b/c need to differentiate overloads, recommend showing lowercase / camelcase arbitrary param name (per Ref docs) rather than the type, in parentheses.